spring-boot-autoconfigure-data-aerospike/pom.xml

```diff
@@ -5,8 +5,8 @@
 
     <parent>
         <groupId>com.aerospike</groupId>
-        <artifactId>spring-boot-data-aerospike</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
+        <artifactId>spring-data-aerospike-starters</artifactId>
+        <version>0.5.0</version>
```

Comment:
I think these can be replaced with ${project.version}

Reply:
I agree,
However, you cant integrate ${project.version} in the children <parent> section so I used "revision" property for that same as described in the second answer here:
https://stackoverflow.com/questions/45598007/properties-in-parent-definition-are-prohibited-in-the-intellij-maven-on-my-mac-o/50448898